### DISCUSSION TITLE
Configuration to lookup specific docker image updates via github releases

### DISCUSSION THREAD

**Discussion by zeldigas** [renovatebot/renovate#20373] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.81.0

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

7.21

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I have a self hosted renovate that is running using official docker image, proxied through Artifactory proxy as `docker.mycompany.com/renovate/renovate`. Renovate is configured to monitor itself and open PR 2 times a month for new versions. It was working but about 3-4 months ago it stopped.

I see in discussions someone mentioned that Artifactory as a proxy might have issues with properly returning all the tags and this can be the reason why PR is not opened for new versions.

I'd like to use github-releases as a source of versions, but fail to find solution other than creating regex manager configuration. Is there alternative more simple option that I failed to find in docs?

### What you tried so far.

I know that I can write a configuration for regex manager and specify datasource there, but is there a better option, like just providing datasource and adjusting dependency name (from `docker.mycompany.org/renovate/renovate` to `renovate/renovate`)

### Relevant debug logs

N/A

**Comment by rarkins** [comment, marked as answer]
I don't think there's any way other than configure `regexManagers` to do this
